students/youjin/westagram/users/views.py

```diff
@@ -1,3 +1,39 @@
-from django.shortcuts import render
+import json, re
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from .models import User
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data              = json.loads(request.body)
+            user_name         = data['name']
+            user_email        = data['email']
+            user_password     = data['password']
+            user_phone_number = data['phone_number']
+            user_gender       = data['gender']
+            user_sns          = data['sns']
+
+            if not re.match('^([a-z0-9_\.-]+)@([\da-z\.-]+)\.([a-z\.]{2,6})$', user_email):  
+                return JsonResponse({"message": "EMAIL_ERROR"}, status=400)
+            
+            if User.objects.filter(email=user_email).exists():
+                return JsonResponse({"message" : "email_alreay_exists"}, status=400)
+            
+            if not re.match('^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{8,}$', user_password):
+                return JsonResponse({"message": "PASSWORD_ERROR"}, status=400)
+
+            users = User.objects.create(
+                name         = user_name,    
+                email        = user_email,
+                password     = user_password,
+                phone_number = user_phone_number,
+                gender       = user_gender,
+                sns          = user_sns,
+            )
```

Comment:
`users` 라는 변수에 생성된 객체를 담을 필요가 없어 보입니다!
```suggestion
            User.objects.create(
                name         = user_name,    
                email        = user_email,
                password     = user_password,
                phone_number = user_phone_number,
                gender       = user_gender,
                sns          = user_sns
            )
```